patches/0001-Specify-the-developer-ID-in-AppStream-meta-data.patch

```diff
@@ -0,0 +1,14 @@
+
+diff --git a/cmake/templates/appdata.xml.in b/cmake/templates/appdata.xml.in
+index f130671..ee774b7 100644
+--- a/cmake/templates/appdata.xml.in
++++ b/cmake/templates/appdata.xml.in
+@@ -9,7 +9,7 @@
+   <url type="homepage">@META_APP_URL@</url>
+   <url type="bugtracker">@META_APP_BUGTRACKER_URL@</url>
+   <launchable type="desktop-id">@META_ID@.desktop</launchable>
+-  <developer><name>@META_APP_AUTHOR@</name></developer>
++  <developer id="org.martchus"><name>@META_APP_AUTHOR@</name></developer>
```

Comment:
this is invalid too, like for the app-id.

Reply:
@Martchus Please help confirm if `io.github.martchus` is suitable for you?

Reply:
It is fine with me.